staging/src/k8s.io/pod-security-admission/test/fixtures_allowPrivilegeEscalation.go

```diff
@@ -38,6 +38,9 @@ func init() {
 			return nil
 		},
 		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			if p.Spec.OS != nil && p.Spec.OS.Name == corev1.Windows {
+				return []*corev1.Pod{p}
```

Comment:
I don't think this is needed any more, but if we do get a windows pod, you'd want to return `[]*corev1.Pod{}`

Reply:
Done.